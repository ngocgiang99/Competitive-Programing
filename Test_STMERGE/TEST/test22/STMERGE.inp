9
4 7
615726805 70646058 675824233 41200971 652730414 895007152 292113789 
773720933 170408626 245533112 788893101 601950395 940260436 536176070 
72857871 682559474 170929994 975930015 299620343 793258478 523106313 
622267208 398555898 514346453 544408902 107316486 678925038 998374156 
8 10
237860583 673493303 896473934 976412694 756827195 307771452 923932867 682619698 791380606 520554560 
104976393 968132342 561760645 931601831 310262039 816714217 342564710 328808727 219976224 42491600 
246534773 72748917 168572826 792287918 330850632 206002661 953758778 812256457 423603379 147788749 
176423669 388743335 280741323 392896721 318028544 138911243 645752294 207506135 977090000 556948732 
584996504 502042116 556842266 949886910 570810806 4700861 103694034 159031510 518494550 661310886 
972191023 397052681 343578061 561922468 715161252 671761245 137161646 268259294 281849831 351799573 
768262608 171339190 884378606 37666744 469069640 786983124 783920108 557944586 589208426 500224283 
934368087 209297168 716458443 596956337 191463395 338247183 691025473 591219707 550963079 27171468 
10 5
212677751 720894284 869136381 935912573 63845159 
4700501 309470766 207767487 173734990 638506986 
766552647 849514185 291759839 932641938 336988510 
158829888 55500816 679277329 707154119 531527944 
900117846 639598956 102085655 158398023 942152294 
368532456 894450468 79383394 861906229 669358859 
379318274 839990013 375416824 213715352 698126086 
137048062 988751139 300095638 305557702 121805706 
585686893 321222472 987147799 261642501 397200841 
210900789 15269388 310813717 671808365 236340769 
4 10
785993166 942495762 957433724 173478598 976254114 135482699 796800096 326405874 98513372 156582482 
283394237 640790470 105526705 163480045 883866345 848092201 145876120 771443488 55307128 867784509 
629718853 463501616 919408447 40761962 621214892 329119138 227634516 876005203 586824708 994376251 
228904715 690739677 194456768 595727616 752958458 900591150 896231574 179635767 286258165 355889323 
5 7
739520785 85583865 483012516 825977419 813380004 565543164 712718214 
142153459 135321154 461709018 228166614 851420576 258559425 413619447 
622485896 798711767 382170933 891175673 736054739 187977565 925600755 
369899717 900459050 279718968 363674513 99522175 466189380 981254827 
788628963 228686989 883554324 655128381 281873663 393520020 794502096 
10 9
645155960 952154093 313875471 778774310 898156639 358367008 337462425 749717916 631076526 
800397406 501120981 86763439 477685513 592689371 922730377 134113972 520845654 888835598 
187362744 141215297 477084089 352244650 463845033 62562346 203126425 651104678 985631456 
785434757 598386840 46193205 602713890 826143654 313710238 659341773 851624052 746970658 
509262286 739147555 865405467 98954447 761386894 940085548 807940508 57635521 44394882 
89315613 90197181 37562702 816155024 71043690 656052098 939465738 273621981 658153674 
841001519 88031946 793423053 808098864 915402571 166556068 898335494 117995219 109908378 
20138732 120349036 851488550 755234850 571497272 121088293 601409503 277514278 594677084 
150283670 952611315 81040173 103493229 171696740 905616023 191705338 416155585 113470835 
892040512 134469524 176367852 45777641 82372514 14801727 90648004 379775786 735872436 
6 1
733806812 
135454122 
295107100 
31517452 
887754980 
36414968 
3 8
707115110 28174164 953310394 813520706 238142152 111302323 393899142 45948082 
342556591 953015114 13494849 846098445 948072703 964903566 236829168 193732839 
703508190 504150580 816566232 85013782 518891952 37448241 889441406 751750830 
7 9
667597991 198875140 600352516 307626051 26130265 641367470 992663440 675109250 927226327 
2225280 62129955 499310957 693839272 356214160 699035490 834139447 980200592 433784370 
826611764 488986198 879202964 781493243 122932156 975128191 425655289 570544321 33454175 
294355878 901865285 599674520 446072516 863880706 430394171 325578710 379236814 682394631 
617756898 327053071 38553762 909984873 293969277 950690584 743243942 888011295 468622483 
642209031 674075519 990995548 453230568 737734704 92068626 24721046 910563202 590091304 
979314051 565255810 340412836 860222661 440992056 758495253 138320596 55671835 333580501 
