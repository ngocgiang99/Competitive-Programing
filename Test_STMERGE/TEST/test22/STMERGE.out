148517457
252765142
63845159
156582482
712718214
133609567
36414968
45948082
35679455
